src/v/storage/tests/log_segment_appender_test.cc

```diff
@@ -39,6 +39,19 @@ namespace {
         std::move(file), segment_appender::options(ss::default_priority_class(), 1));
   }
 
+  iobuf make_random_data(size_t len) {
+        size_t random_len = std::min(1024UL, len);  // NOLINT local constant
+        const auto rbuf = random_generators::gen_alphanum_string(random_len);
+        iobuf output;
+        size_t left = len;
+        while (left > 0) {
+            size_t copy_len = std::min(left, random_len);
+            output.append(rbuf.data(), copy_len);
+            left -= copy_len;
+        }
+        BOOST_CHECK_EQUAL(len, output.size_bytes());
+        return output;
+  }
```

Comment:
can you re-use (or generalize) `random_generators::get_bytes` in src/v/random/generators.h?

Reply:
I will take a look. Less repetition is better.